compiler/modules/front.syntax/ast.bal

```diff
@@ -30,12 +30,10 @@ public type ImportDecl record {|
 |};
 
 public type FunctionDefn record {|
-    *PositionFields;
+    *FunctionTypeDesc;
```

Comment:
This is not the right relationship between a defn and a function type. The position of the end of the type and of the end of the defn are not the same.

Reply:
fixed